### DISCUSSION TITLE
Launch a Thorium window across multiple displays

### DISCUSSION THREAD

**Discussion by nivekxyz** [Alex313031/thorium#42] [Q&A] [answered]
Hey Alex, cool project!

Do you know of a way to get the browser to launch using dimensions that span multiple displays? Currently, using --window-size works to make any size window I want on a single display but if I pass it dimensions out of that display's bounds, it just opens full screen on that one display.

EDIT: I should have specified that this is on Linux with Xorg.

Thanks!

**Comment by Alex313031** [comment, marked as answer]
@nivekxyz I'm sorry but I don't know how you could do that. There is nothing in Thorium that changes anything related to this, so if you can get it to work with Chromium/Chrome, then you should also be able to get it to work with Thorium. If I want to make it span multiple displays I just manually drag the window size, but I know thats not the answer you're looking for since you want it to launch at a predefined px size.

**Reply by Travisnarkle79** [reply]
Neither did I.
